smartswitchplugin/src/main/java/com/kaltura/playkit/plugins/smartswitch/SmartSwitchExecutor.kt

```diff
@@ -134,19 +135,28 @@ internal class SmartSwitchExecutor {
                 connection?.disconnect()
                 log.d("Connection resources have been cleaned.")
             }
+            if (providers == null){
+                providers = ArrayList()
+            }
+            return providers!!
+        }
 
-            return cdnList!!
+        private fun getResponseStringBuilder(inputStream: InputStream?): StringBuilder {
+            val bufferedReader = BufferedReader(InputStreamReader(inputStream))
+            val responseStringBuilder = StringBuilder()
+            var incomingData: String?
+            while (bufferedReader.readLine().also { incomingData = it } != null) {
+                responseStringBuilder.append(incomingData)
+            }
+            return responseStringBuilder
         }
 
         /**
          * Add the Incoming query params.
          */
         private fun appendQueryParams(uri: Uri): Uri {
             val builder: Uri.Builder = uri.buildUpon()
-            builder.appendQueryParameter(accountCodeKey, accountCode)
```

Comment:
Do we not need account code any more ? Please check it because in smart switch config, we are receiving it as a param.

Reply:
Removed from Executor level.